src/pages/Search/SearchPageBottomTab.tsx

```diff
@@ -62,7 +61,7 @@ function SearchPageBottomTab() {
                             activeWorkspaceID={policyID}
                             breadcrumbLabel={translate('common.search')}
                             shouldDisplaySearch={false}
-                            isCustomSearchQuery={shouldUseNarrowLayout && isExpenseFilterPage && queryJSON?.inputQuery !== SearchUtils.buildCannedSearchQuery()}
+                            isCustomSearchQuery={shouldUseNarrowLayout && isExpenseFilterPage && !SearchUtils.isCannedSearchQuery(queryJSON)}
```

Comment:
Why do we want to show the cancel button only for expenses? It should be for any custom query, no?

Reply:
Hmm, @jjcoffee posted [this](https://github.com/Expensify/App/pull/48566#discussion_r1754533309) comment and as such I've made these changes. But happy to revert them